include/boost/beast/experimental/core/url_parser.hpp

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright (c) 2018 jackarain (jack dot wgm at gmail dot com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/beast
+//
+
+#ifndef BOOST_BEAST_EXPERIMENTAL_CORE_URL_PARSER_HPP
+#define BOOST_BEAST_EXPERIMENTAL_CORE_URL_PARSER_HPP
+
+#include <boost/beast/core/string.hpp>
+#include <cctype> // for std::isalpha etc...
```

Comment:
`<cctype>` is dangerous, RFC compliant programs should never be using anything from there.

Reply:
Will remove it, unnecessary with your `isalpha`, `isdigit`, and `ascii_tolower`.

Reply:
Related: https://github.com/boostorg/beast/issues/1232 (something to think about when this is merged)